pkg/systemd/logs.jsx

```diff
@@ -55,36 +55,37 @@ import "./logs.scss";
 const _ = cockpit.gettext;
 
 const timeFilterOptions = [
-    { key: "boot", value: 0, toString: () => _("Current boot"), },
-    { key: "boot", value: "-1", toString: () => _("Previous boot") },
-    { key: "since", value: "-24hours", toString: () => _("Last 24 hours"), default: true },
-    { key: "since", value: "-7days", toString: () => _("Last 7 days") },
+    { key: 1, value: { key: "boot", value: 0 }, content: _("Current boot") },
+    { key: 2, value: { key: "boot", value: "-1" }, content: _("Previous boot") },
+    { key: 3, value: { key: "since", value: "-24hours", default: true }, content: _("Last 24 hours") },
+    { key: 4, value: { key: "since", value: "-7days" }, content: _("Last 7 days") },
```

Comment:
Thanks, that avois the confusion around what `key` is. (A bit weird to use numbers which don't mean anything, but that's fine).